profiles/opentelemetry_dev/README.md

```diff
@@ -0,0 +1,49 @@
+# OpenTelemetry Developer Environment Profile
+
+## Usage
+
+This profile is intended to be used as a developer environment to integrate OpenTelemetry framework with Pulp.
+
+So, the stack is composed by:
+* Prometheus (exposed at port 8889)
+    * Responsible for scraping and storing time-series metrics from the application
+* Grafana (exposed at port 3000)
+    * Used to create visualizations using Prometheus exposed data
+* Jaeger (exposed at port 16686)
+    * To visualize traces from the application
+* OpenTelemetry Collector
+    * Needed to receive, process and export telemetry information between the application and any of its consumers
+
```

Comment:
Let's add an "Examples" section here, with the exact links a user would use to access Granfana/Jaeger assuming all-defaults. And maybe some handholding about what to do once you've accessed the Grafana instance?